packages/component/src/Typography/Typography.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { TypographyProps } from '@blue-learn/schema';
+import React from 'react';
+import Typography from './Typography';
+
+export default {
+	title: 'Example/Typography',
+	component: Typography,
+	argTypes: {
+		value: 'Lorem ipsum dolor sit.',
+	},
+};
+
+const Template = (args: TypographyProps) => (
+	<Typography {...args} />
+);
+
+export const Primary = Template.bind({});
+Primary.args = {
+	value:
+		'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+	fontSize: 'xl',
+	fontWeight: 'bold',
+	textTransform: 'uppercase',
```

Comment:
import interfaces from @blue-learn/schema and use in storybook. do not use strings